prover/server/scripts/test_auth.sh

```
@@ -0,0 +1,42 @@
+curl -s -o /dev/null -w "%{http_code}" -X POST $SERVER_URL/prove -H "Content-Type: application/json" -H "X-API-Key: $API_KEY" -d '{"circuit_type": "inclusion"}'
+echo ""
+
+echo ""
+echo "4. Testing /prove endpoint with Authorization Bearer header (should work if server has API key):"
+curl -s -o /dev/null -w "%{http_code}" -X POST $SERVER_URL/prove -H "Content-Type: application/json" -H "Authorization: Bearer $API_KEY" -d '{"circuit_type": "inclusion"}'
+echo ""
+
+echo ""
+echo "To run this test:"
+echo "1. Set PROVER_API_KEY environment variable: export PROVER_API_KEY=test-api-key-12345"
+echo "2. Start the prover server
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)